virt/virt-2-4-release-notes.adoc

```diff
@@ -100,13 +100,17 @@ from being inadvertently started before an upload has completed.
 [id="virt-2-4-known-issues"]
 == Known issues
 //https://bugzilla.redhat.com/show_bug.cgi?id=1855067 - Requested doc text for this defect.
-//https://bugzilla.redhat.com/show_bug.cgi?id=1835426 - Requested doc text for this defect.
+
+* If a name is specified for a virtual machine instance (VMI) in the VMI template for the virtual machine (VM), it is ignored. The names for the VMI and VM are matched.(link:https://bugzilla.redhat.com/show_bug.cgi?id=1835426[*BZ#1835426*])
```

Comment:
how about `it is now correctly ignored`, to indicate that this is what we want?

Reply:
Reshaped the wording to indicate the behavior applies to this release. Please review.